perf/benchmark/setup_test.sh

```diff
@@ -47,17 +47,17 @@ function run_test() {
   helm -n "${NAMESPACE}" template \
       --set rbac.enabled="${RBAC_ENABLED}" \
       --set includeOutboundIPRanges=$(svc_ip_range) \
-      --set injectL="${LINKERD_INJECT}" \
+      --set client.injectL="${LINKERD_INJECT}" \
```

Comment:
What about server?

If we turn on linkerd don't we want to turn off istio injection?